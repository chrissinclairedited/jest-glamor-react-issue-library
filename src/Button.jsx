import React from "react";
import { css } from "glamor";

export default ({onClick, children}) => (
    <div {...css({fontSize: 10})}><button onClick={onClick}>{children}</button></div>
);
