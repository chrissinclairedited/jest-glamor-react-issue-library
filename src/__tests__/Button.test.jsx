import React from "react";
import { shallow } from "enzyme";
import Button from "../Button";

describe("<Button />", () => {
    it("runs snapshots nicely in the library", () => {
        const component = shallow(<Button onClick={() => {}}>My button</Button>);
        expect(component).toMatchSnapshot();
    });
});
