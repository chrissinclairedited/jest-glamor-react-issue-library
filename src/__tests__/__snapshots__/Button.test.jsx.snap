// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> runs snapshots nicely in the library 1`] = `
.css-0,
[data-css-0] {
  font-size: 10px;
}

<div
  data-css-0=""
>
  <button
    onClick={[Function]}
  >
    My button
  </button>
</div>
`;
